utils/markdown/renderer.jsx

```diff
@@ -171,7 +171,7 @@ export default class Renderer extends marked.Renderer {
         // special case for team invite links, channel links, and permalinks that are inside the app
         let internalLink = false;
         if (this.formattingOptions.siteURL) {
-            const pattern = new RegExp('^' + TextFormatting.escapeRegex(this.formattingOptions.siteURL) + '\\/(?:signup_user_complete|[^\\/]+\\/(?:pl|channels|messages))\\/');
+            const pattern = new RegExp('^' + TextFormatting.escapeRegex(this.formattingOptions.siteURL) + '\\/(?:signup_user_complete|[^\\/]+\\/(?:pl|channels|messages)|admin_console)\\/');
```

Comment:
The `admin_console` should be moved earlier so that this is `?:signup_user_complete|admin_console|...`. Right now, it expects a link like `<mattermost>.com/<some text>/admin_console`

Reply:
I believe `admin_console` is already a peer to `[^\\/]+\\/(?:pl|channels|messages)` and `signup_user_complete` -- but I'm happy to move it up if you think it would be clearer as such. (The redirection does work without `<some text>` preceding `admin_console`.)